netmiko/cisco/cisco_xr.py

```diff
@@ -166,11 +213,13 @@ def exit_config_mode(self, exit_config: str = "end", pattern: str = "") -> str:
             # Read until we detect either an Uncommitted change or the end prompt
             if not re.search(r"(Uncommitted|#$)", output):
                 output += self.read_until_pattern(pattern=r"(Uncommitted|#$)")
```

Comment:
same comment as above. change to full pattern

Reply:
Trying to keep minimal custom changes unless they are absolute necessary.